test/src/unit-cppapi-var-offsets.cc

```diff
@@ -608,15 +660,123 @@ TEST_CASE(
 }
 
 TEST_CASE(
```

Comment:
this is just a missing test for sparse arrays from the previous task of introducing 32-bit offsets.